### PR TITLE
BZ-1698470: Clarified the state of service catalog's custom resources

### PR DESCRIPTION
**Pull request by bergerhoffer**
@openshift/team-documentation For peer review please.

Preview: http://file.rdu.redhat.com/~ahoffer/2019/BZ-1698470/applications/service_brokers/installing-service-catalog.html#sb-install-service-catalog-sb-installing-service-catalog

### CODE REVIEW

**Comment by bergerhoffer** [comment]
@jboyd01 Could you take a look at this information I added and see if it's worded okay? This is for this BZ [1], where there was some confusion as to why there were messages about these resources being removed, when service catalog isn't really installed by default in 4.x.

[1] https://bugzilla.redhat.com/show_bug.cgi?id=1698470

**Comment by jboyd01** [comment]
Good addition @bergerhoffer!
/lgtm

File: modules/sb-install-service-catalog.adoc [hunk 1]
```diff
@@ -7,6 +7,11 @@
 
 If you plan on using any of the services from the {asb-name} or {tsb-name}, you must install the service catalog by completing the following steps.
 
+The custom resources for the service catalog's API server and controller manager
+are created by default in {product-title}, but with a `managementState` of
```
Comment (kalexand-rh) [inline modules/sb-install-service-catalog.adoc]: s/{product-title}, but with/{product-title} with
or 
s/{product-title}, but with/{product-title}, but initially have a

**Review by kalexand-rh — COMMENTED** [review]
I have a suggestion, but it LGTM.

File: modules/sb-install-service-catalog.adoc (hunk 1 above)
Comment (bergerhoffer) [reply modules/sb-install-service-catalog.adoc]: Thanks for the suggestion @kalexand-rh, updated.

**Comment by jianzhangbjz** [comment]
LGTM, thanks!

**Pull request merged**